### PR TITLE
Demographic Summary Report

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: drivers/core_demographics_report/app/controllers/core_demographics_report/warehouse_reports/demographic_summary_controller.rb [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    background_render_action(:render_section, ::BackgroundRender::DemographicSummaryReportJob) do
+      {
+        partial: params.require(:partial).underscore,
+        filters: @filter.for_params[:filters].to_json,
+        user_id: current_user.id,
+      }
+    end
+
+    def index
+      @pdf_export = CoreDemographicsReport::DemographicSummary::DocumentExports::DemographicSummaryExport.new
```
Comment (awisema) [inline drivers/core_demographics_report/app/controllers/core_demographics_report/warehouse_reports/demographic_summary_controller.rb:28]: Redundant w/ `before_action`?

File: drivers/core_demographics_report/app/views/core_demographics_report/warehouse_reports/demographic_summary/index.xlsx.axlsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    bg_color: '81adb9',
+    b: true,
+    border: {style: :thin, color: 'FFFFFF', edges: [:bottom, :top]},
+  }
+)
+wb_styles.add_style(
+  {
+    border: {style: :thin, color: 'FFFFFF', edges: [:bottom, :top]},
+  }
+)
```
Comment (awisema) [inline drivers/core_demographics_report/app/views/core_demographics_report/warehouse_reports/demographic_summary/index.xlsx.axlsx:20]: Accidental dup?

File: drivers/core_demographics_report/app/views/core_demographics_report/warehouse_reports/shared/_disabilities.haml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  - reports << @comparison
+  - column_class = 'col-sm-6'
+.row
+  - reports.each_with_index do |report, index|
+    - comparison_class = ''
+    - if reports.count > 1
+      - comparison_class = "c-swatch__border-left--td-#{index}"
+    %div{class: column_class}
+      %h3 Indefinite and Impairing Disabilities
+      %p The following clients have reported a disability with no subsequent "No" response for that disability, and have reported the disability as Indefinite and impairing.  Clients may have more than one disability type
```
Comment (awisema) [inline drivers/core_demographics_report/app/views/core_demographics_report/warehouse_reports/shared/_disabilities.haml]:
```suggestion
      %p The following clients have reported a disability with no subsequent "No" response for that disability, and have reported the disability as indefinite and impairing.  Clients may have more than one disability type.
```

**Review by awisema — APPROVED** [review]

File: drivers/core_demographics_report/app/views/core_demographics_report/warehouse_reports/demographic_summary/index.xlsx.axlsx (hunk 2 above)
Comment (eanders) [reply drivers/core_demographics_report/app/views/core_demographics_report/warehouse_reports/demographic_summary/index.xlsx.axlsx:20]: Looks like it!

**Pull request merged**